widgets/BigQueryTable-batchsink.json

```diff
@@ -138,6 +138,17 @@
     {
       "label": "Advanced",
       "properties": [
+        {
+          "name": "jobLabelKeyValue",
+          "label": "Job Label Key Value",
+          "widget-type": "keyvalue",
```

Comment:
Is it recommended to set multiple `job labels` for a single BQ job?

Reply:
Adding multiple `job labels` is mentioned in the docs.

>To add multiple labels to a query job, repeat the --label flag and specify a unique key for each label. For example, to add a department:shipping label and cost_center:logistics label

Quoted from docs